.github/workflows/push.yml

```diff
@@ -48,6 +48,37 @@ jobs:
           codecov_yml_path: codecov.yml
           token: ${{ secrets.CODECOV_TOKEN }}
 
+  integration-python:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install Python
+        uses: actions/setup-python@v5
+        with:
+          cache: 'pip'
+          cache-dependency-path: '**/pyproject.toml'
+          python-version: '3.10'
+
+      - name: Install hatch
+        run: pip install hatch==$HATCH_VERSION
+
+      - name: Setup Spark Remote
```

Comment:
Can you create a step in make file as well?

Reply:
it already exists